skycultures/arabian_peninsula/description.en.utf8

```diff
@@ -0,0 +1,109 @@
+<p>The other name <em>The Two Refugees</em> comes from another version of the legend that they are the ones who committed the killing and they fled towards al-Jady for protection.</p>
+<p>These two names are common throughout the central region of Saudi Arabia.</p>
+<h5 id="-l-aba-bzay">Āl Aba Bzay</h5>
+<p>Another name of  <em>The Two Guards</em>, β UMi and γ UMi, used by the tribe of Yam. </p>
+<h5 id="al-khilj-the-first-khilj">Al-Khilj, The First Khilj</h5>
+<p>The two stars α UMa, and β UMa of the Big Dipper that rise first. The name could mean the mother camels. The names are used by Bedouin tribes of central Arabia and the tribe of al-Dawasir in southwest Saudi Arabia </p>
+<h5 id="the-first-two">The First Two</h5>
+<p>A name given to the two stars α UMa, and β UMa, reported in the traditions of west coast of Saudi Arabia.</p>
+<h5 id="the-latter-khilj">The Latter Khilj</h5>
+<p>Names given to the two stars: γ UMa and δ UMa, which are the second pair to rise of the seven stars of the Big Dipper.</p>
+<h5 id="the-fourth-ones">The Fourth ones</h5>
+<p>Same as the <em>Latter Khilj</em>, signifying that they are the third and fourth to rise of the seven stars of the Big Dipper. The name is found in the traditions of west coast of Saudi Arabia.</p>
+<h5 id="mosque-of-al-thurayya">Mosque of al-Thurayya</h5>
+<p>A triangle of stars that rise before al-Thurayya (The Pleiades). It is composed of the three stars: α Ari, α Tri, and β Tri. The source of this name is oral tradition in the region of Wadi al-Dawasir in southwestern Saudi Arabia.</p>
+<h5 id="northern-claw-of-al-jawza-southern-claw-of-al-jawza-claws-of-al-jawza">Northern Claw of al-Jawza, Southern Claw of al-Jawza, Claws of al-Jawza</h5>
+<p>Names of the stars: α, γ, β, and κ Ori, orally communicated from the Qasseem region of Saudi Arabia. The specific name <em>Northern Claw of al-Jawza</em> appeared in the book<em>Hisorty of Najd</em> by the historian Ibn Bishr (died 1873 AD). This might mean the two stars: α, and γ Ori, while Southern claw of al-Jawza would be the two stars: β, and κ Ori</p>
+<h5 id="back-of-al-jawza">Back of al-Jawza</h5>
+<p>The three stars at the belt of Orion. This name along with the star names of Northern Horn of al-Jawza and Southern Horn of al-Jawza comes from the western region of Saudi Arabia and used by the fishers of the red sea. The source is oral tradition from the seaport Umluj of Saudi Arabia.</p>
+<h5 id="the-two-little-dogs">The Two Little Dogs</h5>
+<p>An asterism of the two stars: δ CMa and ε CMa. There heliacal rising is used for timing. The name is common in central Arabia.</p>
+<h3 id="star-names">Star Names</h3>
+<h5 id="the-fifth">The Fifth</h5>

```

Comment:
I think here it would be great to see Arab spelling, English transliteration and English translation.

Reply:
I will implement that if the transliteration standard discussed previously is suitable.